style/swift/README.md

```diff
@@ -5,6 +5,8 @@ Swift
 
 * Keep up with the Objective-C style guide above. Will highlight differences
   here.
+* Prefer `struct`s over `class`es wherever possible
+* If you need to use a `class`, default to marking it as `final`
```

Comment:
The wording of this is sort of based off the previous rule, what about rewording it so it stands on it's own? Something like "Default to marking classes as `final`"?